src/share/util/mct_mod.F90

```diff
@@ -98,6 +98,7 @@ module mct_mod
    use m_GlobalSegMap       ,only: mct_gsMap_gstorage     => GlobalStorage
    use m_GlobalSegMap       ,only: mct_gsMap_ngseg        => ngseg
    use m_GlobalSegMap       ,only: mct_gsMap_nlseg        => nlseg
+   use m_GlobalSegMap       ,only: mct_gsMap_OP           => OrderedPoints
```

Comment:
So remove this line on this branch and when we next merge CIME to ACME, we'll make the required change in CLM.